### PR TITLE
Release 0.4.0

### PR DESCRIPTION
**Pull request by adamjmcgrath**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "start": "rollup -cw",
     "test": "jest --coverage",
     "docs": "typedoc --options typedoc.js src node_modules/@auth0/auth0-spa-js/dist/typings/global.d.ts",
-    "prepack": "npm run test && npm run build && npm run docs"
+    "prepack": "npm run test && npm run build"
```
Comment (adamjmcgrath) [inline package.json:28]: `docs` should be run when doing the release PR

File: package.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
   "author": "Auth0",
   "name": "@auth0/auth0-react",
-  "version": "0.3.1",
-  "description": "Auth0 SDK for React applications",
+  "version": "0.4.0",
+  "description": "Auth0 SDK for React Single Page Applications (SPA)",
```
Comment (adamjmcgrath) [inline package.json:5]: We decided in the channel to specify the SPA bit

File: static/index.html [hunk 3]
```diff
@@ -22,7 +23,7 @@
     ></script>
     <script src="https://cdn.auth0.com/js/auth0-spa-js/1.7/auth0-spa-js.production.js"></script>
     <script src="auth0-react.js"></script>
-    <script type="text/babel">
+    <script type="text/babel" data-presets="es2015,es2016,es2017,stage-3,react">
```
Comment (adamjmcgrath) [inline static/index.html:26]: For testing the playground in IE11

File: package.json (hunk 1 above)
Comment (stevehobbsdev) [reply package.json:28]: In SPA JS this is done through a release script, would be good to automate in the future. Or use one of the `version` hooks if that's what we use.

Comment (adamjmcgrath) [reply package.json:28]: Yeah definitely, will do this in SDK-1575

**Pull request merged**